src/OmniSharp.Host/Startup.cs

```diff
@@ -36,18 +38,21 @@ public Startup(IOmniSharpEnvironment env)
 
             var configBuilder = new ConfigurationBuilder()
                 .SetBasePath(AppContext.BaseDirectory)
-                .AddJsonFile("config.json", optional: true)
+                .AddJsonFile(OmniSharpConstants.OmnisharpConfigFile, optional: true)
                 .AddEnvironmentVariables();
 
             if (env.OtherArgs?.Length > 0)
             {
                 configBuilder.AddCommandLine(env.OtherArgs);
             }
 
+            // Use the global omnisharp config if there's any in the shared path
+            configBuilder.CreateAndAddGlobalOptionsFile(env);
```

Comment:
Other things (not necessarily language servers specificly) do this kind of thing.  If the files don't exist create them.  Atom or VSCode both on start.

From a UX point of view it would be easier to point the user to a file and say "Edit your global settings here".  However since we (omnisharp) aren't really an editor, we could leave that for the editors to implement themselves.